### DISCUSSION TITLE
Youtube issue in Chromium that's not present in Bromite

### DISCUSSION THREAD

**Discussion by yaomtc** [bromite/bromite#1509] [Q&A]
In Chromium (on both my Pixel 4a and my Tab S6 Lite), Youtube videos randomly flicker with looks like one frame of transparent gray over the video, or a variation in brightness of the video. On Bromite, this doesn't happen. Is this an issue that the Bromite team is aware of, and issues in Chromium only get fixed for Bromite? Or might it be a coincidence due to some differences in Bromite?

I tried searching their issues, but I couldn't find one specific to my issue. Should I file one there?

https://bugs.chromium.org/p/chromium/issues/list?q=youtube%20android&can=2

**Comment by csagan5** [comment]
You should try using a different Chromium build and compare, here you should find the version built by Chromium authors: https://storage.googleapis.com/chromium-browser-snapshots/index.html?prefix=Android/905069/

**Comment by yaomtc** [comment]
@csagan5 When I try installing ChromePublic.apk, it asks me if I want to *update* Chromium, which I don't have installed anymore (I'm guessing it sees Vanadium as Chromium?) and when I try to "update", I get "App not installed as package appears to be invalid."

**Reply by yaomtc** [reply]
That was on my Pixel, to clarify. It does install on my Tab, but the flickering issue is there too in their build. I guess I should report it to Google, though I don't know how they wouldn't know about this already.

**Reply by csagan5** [reply]
> I'm guessing it sees Vanadium as Chromium?

It is the other way around: Vanadium uses the package name of Chromium. Chromium is using its own package name.

> It does install on my Tab, but the flickering issue is there too in their build.

In this case you should find the issue on their issue tracker and follow it; most likely it is already there, but if not you can submit a new one.

**Comment by ghost** [comment]
Maybe due to Vulkan being disabled by default in Bromite?
